### PR TITLE
Change color in statistics

### PR DESCRIPTION
**Pull request by Serabe**
2017-02-23 at 15 45

### CODE REVIEW

**Comment by Serabe** [comment]
@marnovo this is the change of color we discussed yesterday. I'm considering incresing the font-weight of attendees / hours / speakers to 300. What do you think?

**Comment by Serabe** [comment]
With `font-weight: 300`:
2017-02-23 at 15 48

**Comment by marnovo** [comment]
@Serabe definitely much better at the weight 300 and with the new brighter blue tone. This way we put less strain on old users' eyes (like mine and Maximo's, for instance). Agreed!

File: _sass/partials/_statistic.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	}
     .stat {
     	font-size: 100px;
-	    color: $link-color;
+	    color: #f4f7fb;
```
Comment (marnovo) [inline _sass/partials/_statistic.scss:9]: Makes sense to have a variable name for this color be quickly changed/reused in other instances, the same way we had previously with `$link-color`? Probably another name would fit.

**Review by marnovo — APPROVED** [review]

**Pull request merged**